app/controllers/api/v1/posters_controller.rb

```diff
@@ -1,7 +1,7 @@
 class Api::V1::PostersController < ApplicationController
   def index
     posters = Poster.all
-    render json: posters #PosterSerializer.format_posters(posters)
+    render json: posters #posters #PosterSerializer.format_posters(posters)
```

Comment:
I will fix this on my next push but saw we have posters twice, one commented out and one we are using. I will take out the comment.